### PR TITLE
Fix up appdata (Issue #5)

### LINKED ISSUE #1: The icon is now missing
Probably something went wrong with the latest update https://flathub.org/apps/details/com.sigmyne.crush

### LINKED ISSUE #5: Fix launchable entry
```
<launchable type="desktop-id">com.sigmyne.crush.:show.desktop</launchable>
```

Is wrong and has some weird characters it should be `$FLATPAK_ID.desktop` exactly. You might need to rename the desktop file to fix this.

### PR DESCRIPTION
**Pull request by attipaci**
- Corrected `<launchable>` entry to `show.desktop`.
- Removed `<translation \>` entry (no translations).
- Removed `<mimetypes>` entry (pulled from manifest).

### CODE REVIEW

**Comment by bbhtt** [comment]
I killed the build, the path in the patch file is wrong

```
can't find file to patch at input line 3
Perhaps you used the wrong -p or --strip option?
The text leading up to this was:
--------------------------
|--- crush.appdata.xml.0	2024-04-23 10:50:12.266087993 +0200
|+++ crush.appdata.xml	2024-04-23 10:55:17.309957323 +0200
--------------------------
```

**Comment by attipaci** [comment]
@bbhtt,

Ooops, I did not pay attention to the paths. I changed them back to what they were. I assume the additional patch timestamps produced y `diff -Naur` do not bother the build...

File: appdata.patch [hunk 1]
```diff
@@ -1,25 +1,31 @@
...
- <mimetypes>
- 	<mimetype>image/fits</mimetype>
---
-2.41.0
-
+-
+-<mimetypes>
+-	<mimetype>image/fits</mimetype>
+-</mimetypes>
++<content_rating>oars-1.1</content_rating>
```
Comment (bbhtt) [inline appdata.patch]:
```suggestion
+<content_rating type="oars-1.1" />
```

**Comment by bbhtt** [comment]
Seems fine.

**Pull request merged**